app/models/agents/telegram_agent.rb

```diff
@@ -33,9 +33,10 @@ class TelegramAgent < Agent
 
       **Options**
 
-      * `caption`: caption for a media content, 0-200 characters
+      * `caption`: caption for a media content (0-200 characters)
       * `disable_notification`: send a message silently in a channel
       * `disable_web_page_preview`: disable link previews for links in a text message
+      * `long_message`: split or truncate text messages or captions that exceeds Telegram API limits
```

Comment:
What about something like below?  I think users would like to know what the default behavior is.
```
set this to `split` (default) or `truncate` to specify how to deal with messages with a long caption or text that exceeds Telegram API limits
```

Reply:
You are right, I will also add a warning about formatting errors.

Currently the default is `truncate` to avoid the involuntary dispatch of multiple messages in abnormal cases, do you think it should be changed to `split`?

Reply:
I don't have a strong opinion about it, but I guess `truncate` sounds reasonable as default behavior.

Reply:
I agree, if we changed the default we would also need to migrate the options of all existing Agents in the database.